modules/ossm-deploy-application-workloads-in-each-mesh.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-in-each-mesh_{context}"]
+= Deploy application workloads in each mesh
+:context: ossm-deploy-application-workloads-in-each-mesh
+
+This procedure demonstrates how to deploy application workloads in separate namespaces.
+
+Run the following command to check for existing {istio} instances.
+[source,terminal]
+----
+$ oc get istios
+----
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Add the `istio-discovery=mesh-1` label to ensure that the namespace is discovered by the first control plane.
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 istio-discovery=mesh-1
+----
+
+. Add the `istio.io/rev=mesh-1` label to the namespace to enable sidecar injection by default while ensuring that pods in this namespace are mapped to the first control plane.
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 istio.io/rev=mesh-1
+----
++
+[NOTE]
+====
+You can verify the `mesh-1` revision name by running the following command:
+[source,terminal]
+----
+$ oc get istiorevisions
+----
+====
+
+. Deploy the `sleep` and `httpbin` applications:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml -n app-ns-1
+----
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml -n app-ns-1
+----
+
+. Wait for the `httpbin` and `sleep` pods to run with sidecars injected:
++
+[source,terminal]
+----
+$ oc get pods -n app-ns-1
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-7f56dc944b-kpw2x   2/2     Running   0          2m26s
+sleep-5577c64d7c-b5wd2     2/2     Running   0          91m
+----
+
+. Create two more namespaces called `app-ns-2` and `app-ns-3`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-2
+----
++
+[source,terminal]
+----
+$ oc create namespace app-ns-3
+----
+
+. Add both namespaces to the second control plane with the discovery selector `istio-discovery=mesh-2` and the revision label with the value `mesh-2`:
```

Comment:
This reads as though we're adding a discovery selector to the namespace, but that's not the case. We're adding a label _that will match the control plane's discovery selector_.